gtk-3.22/gtk-widgets.css

```diff
@@ -1771,25 +1772,51 @@ scrollbar.overlay-indicator slider:hover {
     border-radius: 20px;
 }
 
+scrollbar.bottom trough {
+    border-top-width: 1px;
+}
+
+scrollbar.top trough {
+    border-bottom-width: 1px;
+}
+
+scrollbar.left trough {
+    border-right-width: 1px;
+}
+
+scrollbar.right trough {
+    border-left-width: 1px;
+}
+
 scrollbar trough {
     padding: 0;
+    border-style: solid;
+    border-color: alpha (#000, 0.05);
+}
+
+trough {
+    background-color: @base_color;
+}
+
+scrollbar:hover trough {
+    background-color: transparent;
+    border-radius: 0;
 }
 
 scrollbar.overlay-indicator trough {
     background-color: transparent;
+    border-style: none;
+    border-color: alpha (#000, 0.15);
 }
 
 scrollbar.overlay-indicator:hover trough {
     background-color: alpha (#000, 0.1);
+    border-style: solid;
 }
 
-scrollbar:hover trough {
-    background-color: transparent;
-    border-radius: 0;
-}
-
-trough {
-    background-color: @base_color;
+* > junction {
```

Comment:
wildcard selector is probably not good. Can we do a specific selector here?

Reply:
Are there other uses for junction selector, than junction of crossed scrollbars?
I just decided to kill more rabbits... before they appear :)
Of course, it could be scrolledwindow > junction... or it is all possible use case for junction?

Reply:
Looks like junction is going to remain. Something locks junction style from change (i tried to place this block at the end of gtk-widgets.css).